config/initializers/004-message_bus.rb

```diff
@@ -118,8 +118,16 @@ def setup_message_bus_env(env)
 end
 MessageBus.reliable_pub_sub.max_backlog_size = GlobalSetting.message_bus_max_backlog_size
 
-MessageBus.long_polling_enabled = SiteSetting.enable_long_polling
-MessageBus.long_polling_interval = SiteSetting.long_polling_interval
+if SiteSetting.table_exists? && SiteSetting.where(name: ['enable_long_polling', 'long_polling_interval']).exists?
+  Discourse.deprecate("enable_long_polling/long_polling_interval have switched from site settings to global settings. (Removal after December 2021)")
```

Comment:
What action should be taken when people see this warning? Reset the settings to their default value in the UI? 

Whatever the action is, maybe we should include brief instructions in the message?

Reply:
As annoying as it is, we unfortunately have to deprecate across releases to support people on the stable branch. `Discourse.deprecate` provides the `drop_from` and `since` keyword arguments to make this easier.